src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -42,6 +47,16 @@
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_TUTORIALNAME_T01 = "T01";
```

Comment:
As there are AssessmentTestUtil, AttendanceTestUtil classes, should we create a separate TutorialTestUtil for these variables to be consistent?